provision/provision-site.sh

```diff
@@ -504,4 +516,8 @@ if [ "${SUCCESS}" -ne "0" ]; then
   exit 1
 fi
 
+if [[ $php_version != *"${VVV_BASE_PHPVERSION}"* ]]; then
+  noroot /srv/config/homebin/vvv_restore_php_default
```

Comment:
unsure about the `noroot`, but it looks like one of the commands from my previous feedback! Great minds think alike :)

Reply:
Maybe this check should be inside `vvv_restore_php_default` though?